### PR TITLE
Alpha-version of kubernetes-mesos package.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
cc @jsancio 

TODO:
- [x] add icons
- [x] add post-install instructions
- [x] timestamp (or otherwise version) the image tag, instead of just using `-dev`

### CODE REVIEW

File: repo/packages/K/kubernetes/0/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "tags": ["mesosphere", "framework", "pod", "label", "paas"],
+  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 128 MB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "postInstallNotes": "Kubernetes DCOS Service has been successfully installed!\n\n\tDocumentation: https://github.com/mesosphere/kubernetes-mesos\n\tIssues: https://github.com/mesosphere/kubernetes-mesos/issues",
+  "licenses": [
+    {
+      "name": "Apache License Version 2.0",
+      "url": "https://github.com/mesosphere/kubernetes-mesos/blob/master/LICENSE"
+    }
+  ]
```
Comment (elingg) [inline repo/packages/K/kubernetes/0/package.json]: Can we also add postUninstallNotes here which simply link to our uninstall doc, @jdef?

Comment (jdef) [inline repo/packages/K/kubernetes/0/package.json]: Is there a generic uninstall doc? We don't have one right now for k8s-mesos..

Comment (elingg) [inline repo/packages/K/kubernetes/0/package.json]: It's typically here: http://beta-docs.mesosphere.com/services/chronos/#uninstall or http://beta-docs.mesosphere.com/services/kafka/#uninstall. I see Kubernetes isn't documented at all on our beta-docs site, http://beta-docs.mesosphere.com/services/kubernetes. We need to fill that out, and we have a JIRA open for that.

Comment (jdef) [inline repo/packages/K/kubernetes/0/package.json]: Is it worth dropping a broken link in here for now (that will point to something that exists later)? Or should we submit this PR and follow up with a later one that captures the post-uninstall stuff?

Comment (elingg) [inline repo/packages/K/kubernetes/0/package.json]: I think a follow up PR later will be best to avoid a broken link.

Comment (ConnorDoyle) [inline repo/packages/K/kubernetes/0/package.json]: Agree.

**Comment by ConnorDoyle** [comment]
Thanks @jdef! :ship:ing it.

**Comment by jdef** [comment]
:balloon: :tada: :balloon: :tada: :balloon: :tada: :balloon: :tada: :balloon: :tada: :balloon: :tada:

**Comment by jsancio** [comment]
Oh my! :beers:. Let's get a demo. The team would love to see this.

**Comment by BenWhitehead** [comment]
Great work @jdef!

**Pull request merged**